Makefile

```diff
@@ -315,6 +315,15 @@ azure-secrets:
 	oc create secret generic codecov-token --from-literal=upload=${CODECOV_UPLOAD_TOKEN} -o yaml --dry-run | oc apply -n azure -f -
 .PHONY: azure-secrets
 
+azure4-secrets:
+	oc create secret generic cluster-secrets-azure4 \
+	--from-file=cluster/test-deploy/azure4/osServicePrincipal.json \
```

Comment:
This actually shouldn't be used, can you remove this section?  ci-operator/populate-secrets-from-bitwarden is used instead.